src/Bicep.Cli.IntegrationTests/PublishCommandTests.cs

```diff
@@ -189,7 +189,7 @@ public async Task Publish_ValidFile_ShouldSucceed(DataSet dataSet, string docume
 
         [DataTestMethod]
         [DynamicData(nameof(GetValidDataSets), DynamicDataSourceType.Method, DynamicDataDisplayNameDeclaringType = typeof(DataSet), DynamicDataDisplayName = nameof(DataSet.GetDisplayName))]
-        public async Task Publish_ValidArmTemplteFile_ShouldSucceed(DataSet dataSet)
+        public async Task Publish_ValidArmTemplateFile_ShouldSucceed(DataSet dataSet)
```

Comment:
>Publish_ValidArmTemplateFile_ShouldSucceed [](http://example.com/codeflow?start=26&length=42)

You will need to add more tests to this class to validate the flow. (I also approved CI to run for you, so you should see results soon.)

Reply:
I've added some tests, let me know if I need to change something more.